home_work1/src/test/java/aqa_hw_1/dto/GetPetTests.java

```diff
@@ -0,0 +1,19 @@
+package aqa_hw_1.dto;
+
+import aqa_hw_1.task_1.PetDto;
+import io.restassured.response.Response;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.given;
+
+public class GetPetTests {
+    @Test
+    public void verifyPetCanBeObtained() {
+        Response response = given().get("https://petstore.swagger.io/v2/pet/765");
+        System.out.println(response.asPrettyString());
+        PetDto petDto = response.as(PetDto.class);
+        System.out.println(petDto);
+        Assert.assertEquals(200, response.statusCode());
+    }
```

Comment:
Why is your test located in the dto folder? Dto isn't a test. Make hierarchy structure like in lecture
<img width="457" height="414" alt="Screenshot_40" src="https://github.com/user-attachments/assets/53564d48-df47-43fd-83f4-6a931c330bd0" />